pkg/internal/ebpf/common/http2grpc_transform.go

```diff
@@ -226,7 +224,7 @@ func http2InfoToSpan(info *BPFHTTP2Info, method, path, peer, host string, status
 		Start:         int64(info.StartMonotimeNs),
 		End:           int64(info.EndMonotimeNs),
 		Status:        status,
-		ServiceID:     genericServiceID, // set generic service to be overwritten later by the PID filters
+		ServiceID:     svc.ID{UID: svc.RandomUID(), SDKLanguage: svc.InstrumentableGeneric}, // set generic service to be overwritten later by the PID filters
```

Comment:
I made the UID have initially an UUID, so that the service cache for the trace resource attributes works. I know we replace the service UID immediately in the decorator, but just in case.